test/Renderer/SvgTest.php

```diff
@@ -55,7 +55,7 @@ public function testSvgTransparency()
         $this->renderer->setTransparentBackground(true);
         $this->assertTrue($this->renderer->getTransparentBackground());
         $svgOutput = $this->renderer->draw()->saveXML();
-        $this->assertEquals($svgCompare, $svgOutput);
+        $this->assertEquals(str_replace("\r\n", "\n", $svgCompare), $svgOutput);
```

Comment:
Please use [`assertXmlStringEqualsXmlString`](https://phpunit.readthedocs.io/en/9.0/assertions.html#assertxmlstringequalsxmlstring) instead and then it should work also on Windows.

Reply:
Or maybe even ... `assertXmlStringEqualsXmlFile` ;)

Reply:
Right, we have a file here! 👍

Reply:
If it's available in the min PHPUnit version we are using here (I guess latest v6 because of PHP 5.6).

Reply:
Done, tests pass on PHP 5.6.